From ecccb2fd2ce8c4348e24123b1a7484daf0e8edfa Mon Sep 17 00:00:00 2001
From: Han Gao <gaohan@iscas.ac.cn>
Date: Wed, 12 Nov 2025 22:59:25 +0800
Subject: [PATCH 90/90] debian patch: fix src glx dri common

From: https://salsa.debian.org/xorg-team/lib/mesa/-/blob/mesa-24.0.8-1/debian/patches/src_glx_dri_common.h.diff?ref_type=tags

Signed-off-by: Han Gao <gaohan@iscas.ac.cn>
---
 src/glx/dri_common.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/glx/dri_common.h b/src/glx/dri_common.h
index d762308a4fb..e0e01574991 100644
--- a/src/glx/dri_common.h
+++ b/src/glx/dri_common.h
@@ -57,6 +57,10 @@ extern struct glx_config *driConvertConfigs(const __DRIcoreExtension * core,
 
 extern void driDestroyConfigs(const __DRIconfig **configs);
 
+#ifndef __GLXDRIdrawable
+typedef struct __GLXDRIdrawableRec __GLXDRIdrawable;
+#endif
+
 extern __GLXDRIdrawable *
 driFetchDrawable(struct glx_context *gc, GLXDrawable glxDrawable);
 
-- 
2.47.3

